pkg/crdupdater/namespace.go

```diff
@@ -124,7 +124,15 @@ func (n *Namespace) patch(ns interface{}, isPatched bool) (bool, error) {
 		if len(app) > 0 && len(name) > 0 && len(n.config.version) > 0 {
 			log.Debugf("patch namespace for synopsys label in namespace '%s'", namespace.Name)
 			namespace.Labels[fmt.Sprintf("synopsys.com/%s.%s", app, name)] = n.config.version
-			_, err := util.UpdateNamespace(n.config.kubeClient, namespace)
+
+			getN, err := n.get(namespace.GetName())
+			if err != nil {
+				return false, errors.Annotatef(err, "unable to get the namespace %s", namespace.GetName())
+			}
+			oldNamespace := getN.(*corev1.Namespace)
+			oldNamespace.Labels = namespace.Labels
```

Comment:
Need to be smart here.  Otherwise could end up adding back or removing a label that changed in between original get and this one.  We really need to know what changes we expect to make.

Reply:
done